### ISSUE TITLE
Unable to verify your data submission (while using DbSession)

### ISSUE THREAD

**Issue by pupadupa** [yiisoft/yii2#7696]
I use nginx, PHP 5.5.14, php-fpm, yii2, mac os.

I changed yii config to store session in database (postgress, user is superuser). This is in my config:

```
 'session' => [
        'class' => 'yii\web\DbSession',
        'sessionTable' => 'session',
    ],
```

And now when I try to register new user I've got this error:

```
Bad Request (#400)
Unable to verify your data submission
```

Here is a part of log:

```
10  17:25:57.434    info    yii\db\Command::query   SELECT "data" FROM "session" WHERE "expire">1425993957 AND "id"='cfg9sutufqchr1tdose4cack15'
/Users/pupadupa/Dev/www/mint-office-web/components/Controller.php (41)
11  17:25:57.442    info    yii\web\Session::open   Session started
/Users/pupadupa/Dev/www/mint-office-web/components/Controller.php (41)
12  17:25:57.450    error   yii\web\HttpException:400   exception 'yii\web\BadRequestHttpException' with message 'Не удалось проверить переданные данные.' in /Users/pupadupa/Dev/www/mint-office-web/vendor/yiisoft/yii2/web/Controller.php:110
Stack trace:
#0 /Users/pupadupa/Dev/www/mint-office-web/components/Controller.php(41): yii\web\Controller->beforeAction(Object(app\controllers\user\RegistrationAction))
#1 /Users/pupadupa/Dev/www/mint-office-web/vendor/yiisoft/yii2/base/Controller.php(149): app\components\Controller->beforeAction(Object(app\controllers\user\RegistrationAction))
#2 /Users/pupadupa/Dev/www/mint-office-web/vendor/yiisoft/yii2/base/Module.php(455): yii\base\Controller->runAction('registration', Array)
#3 /Users/pupadupa/Dev/www/mint-office-web/vendor/yiisoft/yii2/web/Application.php(83): yii\base\Module->runAction('user/registrati...', Array)
#4 /Users/pupadupa/Dev/www/mint-office-web/vendor/yiisoft/yii2/base/Application.php(375): yii\web\Application->handleRequest(Object(yii\web\Request))
#5 /Users/pupadupa/Dev/www/mint-office-web/web/index.php(20): yii\base\Application->run()
#6 {main}
13  17:25:57.454    trace   yii\base\Controller::runAction  Route to run: index/error
```

BTW,
1. I've got `<?= Html::csrfMetaTags() ?>` in head section and I have csrf input in my form. So it seems not the problem.
2. I don't want to do public $enableCsrfValidation = false; because I think it's not solution but workaround.
   How could I understand what cause this error? As I mentioned earlier problem only appears when I store session in database.

I could set and get variable from session. For example, I put it in beoreAction of Controller.php

```
Yii::$app->session->set('test', 'qwe');
$t = Yii::$app->session->get('test') ;
var_dump($t);
```

But after that If I comment first line like this

```
//Yii::$app->session->set('test', 'qwe');
$t = Yii::$app->session->get('test') ;
var_dump($t);
```

and refresh the page - I recieve NULL (BTW I could see Cookie:PHPSESSID=cfg9sutufqchr1tdose4cack15 in cookies after refresh).

So it seems there are some problems with session (DbSession) or maybe my php/php-fpm/nginx settings.

I've post some additional information on SO http://stackoverflow.com/questions/28965388/yii2-unable-to-verify-your-data-submission

**Comment by samdark** [comment]
https://github.com/yiisoft/yii2/issues/7692

**Comment by samdark** [comment]
https://github.com/yiisoft/yii2/issues/6907

**Comment by samdark** [comment]
Should be fixed by #14542
